tasks/release-notes-only.yml

```diff
@@ -0,0 +1,16 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source: {repository: concourse/unit}
+
+inputs:
+- name: concourse-pr
+
+run:
+  path: grep
+  args:
+  - -v
+  - release-notes
+  - concourse-pr/.git/resource/changed_files
```

Comment:
Just to make sure, if new documentation files are added or deleted, they're also included as changed files?

Reply:
I'm pretty sure that's how it works - instinctively I'd expect the `changed_files` file to have something like the output of `git diff master --name-only --pretty=''` based on what we see [here](https://github.com/telia-oss/github-pr-resource#get). We haven't actually tested this, because it didn't seem worth the overhead of setting up a fake repo and fake PRs.